### DISCUSSION TITLE
Short Key for Hide/Show Side Bar

### DISCUSSION THREAD

**Discussion by crazyoptimist** [Eugeny/tabby#2946] [General]
Would be very appreciated if I can have such short key and/or ui button.
Thanks for your hard work @Eugeny and all contributors.

**Comment by Eugeny** [comment]
What side bar?

**Reply by crazyoptimist** [reply]
![image](https://user-images.githubusercontent.com/55074937/91584149-d842d680-e917-11ea-8738-f6c87ce47ef3.png)
Here you go.

**Comment by Eugeny** [comment]
You're looking for Termius, not Terminus.

**Reply by crazyoptimist** [reply]
Sorry!
